src/App.tsx

```diff
@@ -16,47 +46,78 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState(SortType.Default);
+  const [isReversed, setIsReversed] = useState(false);
+  const visibleGoods = getOrderedGoods(
+    goodsFromServer, sortField, isReversed,
+  );
+  const isSorted = !!(sortField || isReversed);
+  const setDefault = () => {
```

Comment:
We need to choose a better name for this handler:
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364